build-system/tasks/runtime-test.js

```diff
@@ -28,22 +28,37 @@ var webserver = require('gulp-webserver');
 var app = require('../test-server').app;
 var karmaDefault = require('./karma.conf');
 
+
+/**
+ * Logs a message if tests are being run during local development.
+ */
+function logLocal() {
+  if (!process.env.TRAVIS) {
+    util.log(util.colors.green(...arguments));
```

Comment:
@jridgewell, do you know if there's a node 4 equivalent for the spread operator? (See https://travis-ci.org/ampproject/amphtml/jobs/261941064)